recipes/p4p/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "p4p" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mdavidsaver/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 2112c7fb851c80cd7b9b8ed065265a510fef1376c0aaf9e1e8fbdeca77b7e2c7
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - epicscorelibs
+    - numpy
+    - pip
+    - setuptools
+    - setuptools_dso
+  run:
+    - python
+    - epicscorelibs
+    - numpy
```

If you are compiling against ``numpy`` it should be 
```suggestion
    - {{ pin_compatible('numpy') }}
```